api/adobe_vendor_id.py

```diff
@@ -162,91 +133,6 @@ def device_id_handler(self, device_id):
-<label>%(label)s</label>
-</accountInfoResponse>"""
-
-    AUTH_ERROR_TYPE = "AUTH"
-    ACCOUNT_INFO_ERROR_TYPE = "ACCOUNT_INFO"
-
-    ERROR_RESPONSE_TEMPLATE = '<error xmlns="http://ns.adobe.com/adept" data="E_%(vendor_id)s_%(type)s %(message)s"/>'
-
-    TOKEN_FAILURE = 'Incorrect token.'
-    AUTHENTICATION_FAILURE = 'Incorrect barcode or PIN.'
-    URN_LOOKUP_FAILURE = "Could not identify patron from '%s'."
-
-    def __init__(self, vendor_id):
-        self.vendor_id = vendor_id
-
-    def handle_signin_request(self, data, standard_lookup, authdata_lookup):
-        parser = AdobeSignInRequestParser()
-        try:
-            data = parser.process(data)
-        except Exception as e:
-            logging.error("Error processing %s", data, exc_info=e)
-            return self.error_document(self.AUTH_ERROR_TYPE, str(e))
-        user_id = label = None
-        if not data:
-            return self.error_document(
-                self.AUTH_ERROR_TYPE, "Request document in wrong format.")
-        if not 'method' in data:
-            return self.error_document(
-                self.AUTH_ERROR_TYPE, "No method specified")
-        if data['method'] == parser.STANDARD:
-            user_id, label = standard_lookup(data)
-            failure = self.AUTHENTICATION_FAILURE
-        elif data['method'] == parser.AUTH_DATA:
-            authdata = data[parser.AUTH_DATA]
-            user_id, label = authdata_lookup(authdata)
-            failure = self.TOKEN_FAILURE
-        if user_id is None:
-            return self.error_document(self.AUTH_ERROR_TYPE, failure)
-        else:
-            return self.SIGN_IN_RESPONSE_TEMPLATE % dict(
-                user=user_id, label=label)
-
-    def handle_accountinfo_request(self, data, urn_to_label):
-        parser = AdobeAccountInfoRequestParser()
-        label = None
-        try:
-            data = parser.process(data)
-            if not data:
-                return self.error_document(
-                    self.ACCOUNT_INFO_ERROR_TYPE,
-                    "Request document in wrong format.")
-            if not 'user' in data:
-                return self.error_document(
-                    self.ACCOUNT_INFO_ERROR_TYPE,
-                    "Could not find user identifer in request document.")
-            label = urn_to_label(data['user'])
-        except Exception as e:
-            return self.error_document(
-                self.ACCOUNT_INFO_ERROR_TYPE, str(e))
-
-        if label:
-            return self.ACCOUNT_INFO_RESPONSE_TEMPLATE % dict(label=label)
-        else:
-            return self.error_document(
-                self.ACCOUNT_INFO_ERROR_TYPE,
-                self.URN_LOOKUP_FAILURE % data['user']
-            )
-
-    def error_document(self, type, message):
-        return self.ERROR_RESPONSE_TEMPLATE % dict(
-            vendor_id=self.vendor_id, type=type, message=message)
-
-
 class DeviceManagementRequestHandler(object):
```

Comment:
@killerpuppytails And this one too, is this no longer in use?

Reply:
Per our Slack convo, since Adobe is a black box, please do an initial PR with this intact, and we'll remove this one in the PR after the next one.